tests/sys/src/TestFileSystem.hx

```diff
@@ -94,6 +94,11 @@ class TestFileSystem extends utest.Test {
 		}
 	}
 
+	function testRootExists() {
+		Assert.isTrue(FileSystem.exists("/"));
```

Comment:
Should we also test drive letter on windows as mentioned in the start message of #6489?

Reply:
There already is a test for that in a separate function. It tests for `C:`, but the function treats `D:` no differently.